lib/libesp32/berry_matter/src/embedded/Matter_Plugin_3_Light1.be

```diff
@@ -168,6 +168,9 @@ class Matter_Plugin_Light1 : Matter_Plugin_Light0
         return tlv_solo.set(TLV.U1, 0)    #
       elif attribute == 0x0011          #  ---------- OnLevel / u1 ----------
         return tlv_solo.set(TLV.U1, self.shadow_bri)
+      
+      elif attribute == 0xFFFC          #  ---------- FeatureMap / map32 ----------
+        return tlv_solo.set(TLV.U4, 0x01)    # LT
```

Comment:
Featuremaps are managed centrally in `Matter_Plugin_0.be` in the map `FEATURE_MAPS`.
Please move it there instead (this is meant to limit the size of code and avoid code duplication)